doc/rtd/explanation/format.rst

```diff
@@ -24,93 +59,132 @@ These things may include:
 - importing certain SSH keys or host keys
 - *and many more...*
 
-See the :ref:`yaml_examples` section for a commented set of examples of
-supported cloud config formats.
+Many modules are available to process cloud-config data. These modules
+may run once per instance, every boot, or once ever. See the associated
+module to determine the run frequency.
 
-Begins with: ``#cloud-config`` or ``Content-Type: text/cloud-config`` when
-using a MIME archive.
-
-.. note::
-   Cloud config data can also render cloud instance metadata variables using
-   :ref:`jinja templates <instancedata-Using>`.
+See the :ref:`yaml_examples` section for a set of commented examples of
+supported cloud config formats.
```

Comment:
> set of commented examples of supported cloud config formats.

I know that you didn't write this in this PR, but I don't think that it makes sense to call these "cloud config formats". 

Also, this seems like an appropriate place to link to the modules reference page as well. Could we do that while making this change?

How about something like this:
```rst
For more information, see the cloud config 
:ref:`example configurations page <link_to_examples_page>`
 or the cloud config :ref:`modules reference page<link_to_modules_page>`.
```